conwayste/src/ui/context.rs

```diff
@@ -93,9 +95,27 @@ pub enum EventType {
 #[derive(Debug, Clone)]
 pub struct Event {
     pub what: EventType,
-    pub point: Point2<f32>,
+    pub point: Point2<f32>, // Must not be None if this is a mouse event type
```

Comment:
Does the comment actually apply here? Seems confusing as there isn't an `Option`.